lms/djangoapps/discussion/rest_api/tests/test_serializers.py

```diff
@@ -760,7 +772,9 @@ def test_create_endorsed(self):
             'course_id': [str(self.course.id)],
             'body': ['Test body'],
             'user_id': [str(self.user.id)],
-            'endorsed': ['True']
+            'endorsed': ['True'],
+            'anonymous': ['False'],
+            'anonymous_to_peers': ['False'],
         }
```

Comment:
Can you also test serializer with anonymous=True, anonymous_to_peers=True?

Reply:
Pushed a commit with new tests. Let me know if that's not what you meant.